src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,129 @@
+package homework_2.random_chars_table;
+
+import lombok.Getter;
+import lombok.Setter;
+import java.util.*;
+
+@Getter
+@Setter
+public class RandomCharsTable {
+
+    public enum Strategy {
+        EVEN,
+        ODD
+    }
+    private int length;
+    private int width;
+    private Strategy strategy;
+    private int[][] table;
+
+    public void run() {
+        Scanner scanner = new Scanner(System.in);
+        String[] arr = scanner.nextLine().trim().split("\\s");
+
+        scanner.close();
+        try {
+            setParameters(arr);
+        } catch (NumberFormatException ex) {
+            System.out.println("Passed parameters should match the format [positive integer] [positive integer] [even|odd]!");
+            return;
+        } catch (IllegalArgumentException ex) {
+            System.out.println("Passed parameters should match the format [positive integer] [positive integer] [even|odd]!");
+            return;
+        }
```

Comment:
```suggestion
        } catch (IllegalArgumentException ex) {
            System.out.println("Passed parameters should match the format [positive integer] [positive integer] [even|odd]!");
            return;
        }
```
Так как `IllegalArgumentException` является родителем `NumberFormatException`

Reply:
Но если бы сообщения об ошибке были бы разными, это было бы существенно?